docs/root/install/building.rst

```diff
@@ -32,7 +32,20 @@ Pre-built binaries
 ------------------
 
 On every master commit we create a set of lightweight Docker images that contain the Envoy
-binary. We also tag the docker images with release versions when we do official releases.
+binary.
+
+* `envoyproxy/envoy-dev <https://hub.docker.com/r/envoyproxy/envoy-dev/tags/>`_: Release binary with
+  symbols stripped on top of an Ubuntu Xenial base.
```

Comment:
Can you remind me why we have stripped binaries for some Docker images and not for others?

Reply:
Size reasons.